READMEs/compute-flow.md

```diff
@@ -78,7 +83,7 @@ from ocean_lib.data_provider.data_service_provider import DataServiceProvider
 
 #Alice's config
 config = {
-   'network' : 'rinkeby',
+   'network' : os.getenv("OCEAN_NETWORK_URL"),
```

Comment:
Should I keep the same logic here and replace only the envvar, @trentmc?

Reply:
> Should I keep the same logic here and replace only the envvar, @trentmc?

Yes, of course